examples/benchmark/tls_bench.c

```diff
@@ -1368,12 +1368,11 @@ int bench_tls(void* args)
     }
     else {
         /* Run for each cipher */
-        const int ciphersSz = 4096;
-        ciphers = (char*)XMALLOC(ciphersSz, NULL, DYNAMIC_TYPE_TMP_BUFFER);
+        ciphers = (char*)XMALLOC(WOLFSSL_CIPHER_LIST_MAX_SIZE, NULL, DYNAMIC_TYPE_TMP_BUFFER);
         if (ciphers == NULL) {
             goto exit;
         }
-        wolfSSL_get_ciphers(ciphers, ciphersSz);
+        wolfSSL_get_ciphers(ciphers, (int)sizeof(ciphers));
```

Comment:
sizeof(ciphers) does not look right.

Reply:
Used the sizeof() for consistency. Technically it was correct. Changed it to use the macro which saves one function call.